app/views/collections/edit.html.erb

```diff
@@ -1,6 +1 @@
-<h1>Editing Collection</h1>
```

Comment:
Drop a consistent `<h1>` back at this level. Give it the `sr-only` class if you don't want to display it.